src/components/home/Landing.jsx

```diff
@@ -7,17 +7,17 @@ import Link from "next/link";
 const Landing = () => {
   return (
     <div className="flex h-screen ml-6 overflow-x-clip">
-      <div className="flex flex-col w-8/12 justify-center gap-3">
+      <div className="flex flex-col w-8/12 justify-center gap-3 md:gap-0">
         <div className="flex justify-center">
           <Image src={logo} alt="rishi-logo" className="w-24 h-auto" />
         </div>
 
-        <div className="flex justify-center font-urbanist font-extrabold text-rishi-black text-5xl">
+        <div className="flex justify-center font-urbanist font-extrabold text-rishi-black text-4xl md:text-5xl">
```

Comment:
can you center this text, the subtext is centered but this is left aligned for some reason